e2e-test/src/test/features/BigQueryToDeduplicate.feature

```diff
@@ -0,0 +1,109 @@
+    Then Connect plugins: "BigQuery" and "Deduplicate" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter input plugin property: "datasetProjectId" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqSourceTable"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "deduplicateOutputSchemaBQ"
+    Then Validate "BigQuery" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "Deduplicate"
+    Then Enter Deduplicate plugin property: filterOperation field name with value: "deduplicateFilterFieldName"
+    Then Select Deduplicate plugin property: filterOperation field function with value: "deduplicateFunction"
+    Then Select dropdown plugin property: "uniqueFields" with option value: "fname"
+    Then Press ESC key to close the unique fields dropdown
+    Then Select dropdown plugin property: "uniqueFields" with option value: "lname"
+    Then Press ESC key to close the unique fields dropdown
+    Then Validate "Deduplicate" plugin properties
+    Then Close the Plugin Properties page
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "Deduplicate" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery2"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter input plugin property: "datasetProjectId" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Validate "BigQuery2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count of deduplicate is equal to IN record count of sink
```

Comment:
we should validate that the data is written to the sink is correct. i.e. deduplicate plugin worked correctly

Reply:
As per the changes suggested in https://github.com/cdapio/hydrator-plugins/pull/1644 included plugins from hydrator-plugin repo only. Hence removed the scenario.